### DISCUSSION TITLE
Inline Encryption

### DISCUSSION THREAD

**Discussion by pbl987** [anonaddy/anonaddy#159] [Q&A] [answered]
Is there a way to send the encrypted mails as inline encrypted instead as attachment?
I havent found a way to do so, but i need it for the Firefox plugin Mailvelope, it doesnt work otherwise.

**Comment by willbrowningme** [comment, marked as answer]
Not at the moment no, but it's a good idea. I have used Mailvelope in the past with Tutanota so understand your issue. I'll have a look and see if I can add a setting to enable this.

You can still download the encrypted part of the message in Mailvelope and then manually paste in the encrypted text to decrypt it, even though it is a bit of a pain.
